README.md

```diff
@@ -35,59 +43,74 @@ The MixedReality-SpectatorView repository uses Unity packages, git submodules an
 
 After going through the setup steps in 'Obtaining the code' and 'Setting up your local environment', sample projects will be configured for use in your clone of the MixedReality-SpectatorView repository. It's easy to get started by building off one of the samples or by inspecting them to understand project setup. For more information, see [Samples](samples/README.md).
 
-## Setting up your own project
-
 ### Adding references to your own project
 
 After obtaining a local clone of the MixedReality-SpectatorView repository and resolving its external dependencies (see above), the suggested mechanism for referencing the code is by adding symbolic linked directories to your Unity project's Assets folder. You can do this with the following:
 
+> Note: Symbolic linked directories should be setup as relative paths. Using relative paths should allow directories to resolve correctly regardless of where you or your team members clone your project repository in their local file systems. The instructions below demonstrate setting up symbolic linked directories based on the following paths:
+>* **Project repository directory:** c:\Your\Unity\Project
+>* **Project Assets directory:** c:\Your\Unity\Project\Assets
+>* **MixedReality-SpectatorView submodule directory:** c:\Your\Unity\Project\sv
+
 #### Using the `release/1.0.0-beta` branch
 
-1) Close any instance of Unity.
+1) Close any instances of Unity.
 2) Open an administrator command window.
 3) Run the following commands, updating the paths to reflect your local environment:
 
-* `cd c:\Your\Unity\Project\Assets`
-* `set PathToRepo="c:\your\path\to\MixedReality-SpectatorView\"`
-* `mklink /D "MixedReality-SpectatorView" "%PathToRepo%\MixedReality-SpectatorView\src\SpectatorView.Unity\Assets"`
-* `mklink /D "ARKit-Unity-Plugin" "%PathToRepo%\MixedReality-SpectatorView\external\ARKit-Unity-Plugin"`
-* `mklink /D "AzureSpatialAnchorsPlugin" "%PathToRepo%\MixedReality-SpectatorView\external\Azure-Spatial-Anchors-Samples\Unity\Assets\AzureSpatialAnchorsPlugin"`
-* `mklink /D "GoogleARCore" "%PathToRepo%\MixedReality-SpectatorView\external\ARCore-Unity-SDK\Assets"`
-* `mklink /D "MixedReality-QRCodePlugin" "%PathToRepo%\MixedReality-SpectatorView\external\MixedReality-QRCodePlugin"`
-* `mkdir AzureSpatialAnchors.Resources`
-* `cd AzureSpatialAnchors.Resources`
-* `mklink /D "android-logos" "%PathToRepo%\MixedReality-SpectatorView\external\Azure-Spatial-Anchors-Samples\Unity\Assets\android-logos"`
-* `mklink /D "logos" "%PathToRepo%\MixedReality-SpectatorView\external\Azure-Spatial-Anchors-Samples\Unity\Assets\logos"`
+4. `cd c:\Your\Unity\Project\Assets`
+5. `mklink /D "MixedReality-SpectatorView" "..\sv\src\SpectatorView.Unity\Assets"`
+6. `mklink /D "ARKit-Unity-Plugin" "..\sv\external\ARKit-Unity-Plugin"`
+7. `mklink /D "AzureSpatialAnchorsPlugin" "..\sv\external\Azure-Spatial-Anchors-Samples\Unity\Assets\AzureSpatialAnchorsPlugin"`
+8. `mklink /D "GoogleARCore" "..\sv\external\ARCore-Unity-SDK\Assets"`

```

Comment:
Nit: Can we also use the same folder name (i.e. ARCore-Unity-SDK) instead of GoogleARCore?

I see that there is a subfolder called GoogleARCore, and when I saw this during my attempt to get this setup, I wasn't sure if the intent was to take the entire folder, or just that subfolder

Reply:
Good catch, we actually want to point to the GoogleARCore folder hence the name. The above line is just wrong.